.github/scripts/trymerge.py

```diff
@@ -736,11 +759,19 @@ def merge_into(self, repo: GitRepo, *, force: bool = False, dry_run: bool = Fals
         else:
             self.merge_ghstack_into(repo, force, comment_id=comment_id)
 
-        repo.push(self.default_branch(), dry_run)
+    def create_land_time_check_branch(self, repo: GitRepo) -> str:
+        branch_name = f'landchecks/{self.pr_num}'
+        try:
+            repo._run_git('branch', "-D", branch_name)
+        except Exception:
+            pass
+        repo._run_git('checkout', "-b", branch_name)
+        repo._run_git('push', '-u', 'origin', branch_name, '--force')
+        commit = repo.get_commit('HEAD').commit_hash
         gh_post_pr_comment(self.org, self.project, self.pr_num,
-                           f"@{self.get_pr_creator_login()} your PR has been successfully merged.", dry_run)
-        if not dry_run:
-            gh_add_labels(self.org, self.project, self.pr_num, ["merged"])
+                           'Successfully started land time checks.' +
+                           f' See progress here: https://github.com/{self.org}/{self.project}/commit/{commit}')
```

Comment:
Would maybe the hud view be a better link here to track signals?

Reply:
That's a good point. The GH UI actually links the commit that was created in the PR.